packages/lexical-playground/src/plugins/ToolbarPlugin.tsx

```diff
@@ -975,6 +1000,13 @@ export default function ToolbarPlugin(): JSX.Element {
 
   return (
     <div className="toolbar">
+      <button
+        onClick={clearFormatting}
+        title="Clear Formatting"
```

Comment:
Might be a little clearer for accessibility: 
```suggestion
        title="Clear text formatting"
```